ui/src/components/TaskGroupStats/index.jsx

```diff
@@ -0,0 +1,275 @@
+import React, { Component } from 'react';
+import { shape, arrayOf, string } from 'prop-types';
+import { Paper, Grid, withStyles } from '@material-ui/core';
+import { taskState, pageInfo, task } from '../../utils/prop-types';
+import { THEME } from '../../utils/constants';
+import Link from '../../utils/Link';
+import {
+  formatTime,
+  sampleTasks,
+  quantile,
+  filterTasksWithDuration,
+} from '../../utils/task';
+
+@withStyles(theme => ({
```

Comment:
You're a patient man Yarik! Styling looks great in the UI screenshots you provided 👍🏻 Nice!

Reply:
now as you said this, I noticed that not everything is aligned 😂 
re-pushed with proper padding